src/DynamoCoreWpf/ViewModels/Watch3D/shaderSource/psDynamoMesh.hlsl

```diff
@@ -0,0 +1,127 @@
+
+#define MESH
+#include"Common.hlsl"
+#include"CommonBuffers.hlsl"
```

Comment:
These files are in a nested folder called `helix_shader_defs` - does it know how to find them?

Reply:
yes - note the `/I ./helix_shader_defs/COMMON` flag in the compile script.